src/ne2k.js

```diff
@@ -125,7 +125,9 @@ function Ne2k(cpu, bus)
         this.memory[i << 1] = this.memory[i << 1 | 1] = mac[i];
     }
 
-    this.memory[14] = this.memory[15] = 0x57;
+    for(var i = 28; i < 32; i++) {
```

Comment:
Insert newline before `{`

Reply:
See updated, removed the loop and added comment for clarity, think that may be better to explain what's going on.